### PR TITLE
[docs] adjust zklogin intro

### PR DESCRIPTION
**Pull request by sblackshear**
- Tighten up the first paragraph
- Make the point titles and text punchier
- Emphasize 2FA feature
- Add bullet for security
- Clarify verified identity feature

### CODE REVIEW

File: doc/src/build/zk_login.md [hunk 1]
```diff
@@ -2,23 +2,24 @@
...
+1. **Private**: Zero knowledge proofs prevent third parties from linking a Sui address with its corresponding OAuth identifier.
 
-1. **Natively Supported**: zkLogin is a protocol-level signature scheme, benefiting from [crypto agility](https://docs.sui.io/devnet/learn/cryptography/sui-signatures). Unlike smart contract-based alternatives, it will smoothly integrate with sponsored transactions and MultiSig.
+1. **Optional Verified Identity**: However, a user can *opt in* to verify the OAuth identifier that was used to derive a particular Sui address. This serves as the foundation for a verifiable on-chain identity layer.
 
-1. **Foundation for the Identity Layer**: In the future, zkLogin can serve as an opt-in identity layer for users on-chain.
+1. **Accessible**: zkLogin is one of several native Sui signature schemes thanks to Sui's [crypto agility](https://docs.sui.io/devnet/learn/cryptography/sui-signatures). It smoothly integrates with other powerful Sui primitives like sponsored transactions and multisig.
 
-Are you a builder who wants to integrate with zkLogin into your wallet or application? Dive into our [Integration guide](#integration-guide).
+
```
Comment (sblackshear) [inline doc/src/build/zk_login.md:21]: @kchalkias, I'd like to add one more point called something like "Built with rigor" that will let us link to the proofs, audits, ceremony. The goal of this is to make the reader aware of the expertise and care that went into creating this. Could you help me suggest something?

File: doc/src/build/zk_login.md [hunk 2]
```diff
@@ -2,23 +2,24 @@
...
+zkLogin is designed with the following goals in mind:
 
-zkLogin is designed with the following principles in mind:
+1. **Simple Onboarding**: zkLogin allows users transact on Sui using the familiar OAuth login flow, eliminating the friction of handling cryptographic keys or remembering mnemonics.
 
-1. **Simple Onboarding**: With the familiar OAuth login flow, zkLogin enables users to easily create a Sui wallet and engage with Sui—eliminating the need for cryptographic key management.
+1. **Fully Self-Custodial**: A zkLogin transaction requires user approval via the standard OAuth login process--the OAuth provider cannot transact on the user's behalf.
 
-1. **Fully Self-Custodial**: A zkLogin transaction requires user approval within the OAuth login process, preventing unilateral fund movement from a zkLogin wallet—even by the OAuth provider.
+1. **Secure**: zkLogin is a two-factor authentication scheme: sending a transaction requires both a credential from a recent OAuth login and a password not managed by the OAuth provider. An attacker who compromises an OAuth account cannot transact from the user's corresponding Sui address unless they separately compromise the password.
```
Comment (kchalkias) [inline doc/src/build/zk_login.md:13]: we generally don't use the word password (which is typically something user generated and with high probability lower entropy than 128 bits), instead we use the wording: high entropy salt per account.

File: doc/src/build/zk_login.md (hunk 1 above)
Comment (kchalkias) [reply doc/src/build/zk_login.md:21]: This is what we sent to ceremony participants to ensure security, audit and transparency.

If you are interested in what happens in the background during the ceremony, see https://sui.io/zklogin for more and FAQ.

Also, please feel free to review the open-source code repositories and documentation:

- Repo links:
    - Circuit: https://github.com/sui-foundation/zklogin-circuit
    - Ceremony: https://github.com/sui-foundation/zk-ceremony-client
- Doc links:
    - Developer docs: https://docs.sui.io/build/zk_login
    - Audit report from zkSecurity: https://github.com/sui-foundation/security-audits/blob/main/zksecurity_zklogin-circuits.pdf

**Review by kchalkias — APPROVED** [review]
small comments

File: doc/src/build/zk_login.md (hunk 1 above)
Comment (kchalkias) [reply doc/src/build/zk_login.md:21]: TL;DR:
- Credibility / Security: We allowed two software stacks for ZK ceremony contributions, there were more than 100 participants, many of them notable for their contribution in the general tech, security and blockchain community.
- Transparency: All related code has been open-sourced.
- Security: Both the ceremony and zklogin related code has been audited by at least two auditors.
- Innovation + Rigorousness: We have multiple unit tests, crosschecked and contributed in multiple open source projects related to Groth16 stack, in order to ensure compatibility and consistency.

File: doc/src/build/zk_login.md (hunk 2 above)
Comment (sblackshear) [reply doc/src/build/zk_login.md:13]: This is probably a broader conversation, but I I think it's really important to use a familiar term like "password" even if from a cryptography perspective "high entropy salt" is the precise description. IMO we need a commonsense term that captures how devs/users should think about and store these.

Comment (sblackshear) [reply doc/src/build/zk_login.md:13]: This is probably a broader conversation, but I I think it's really important to use a familiar term like "password" even if from a cryptography perspective "high entropy salt" is the precise description. IMO we need a commonsense term that captures how devs/users should think about, store, and describe these.

Comment (sblackshear) [reply doc/src/build/zk_login.md:13]: I'll use "salt" for now pending a broader discussion

File: doc/src/build/zk_login.md [hunk 3]
```diff
@@ -2,23 +2,25 @@
 title: About zkLogin
 ---
 
-zkLogin is a Sui primitive that lets wallets and apps link Sui objects with an OAuth credential—enabling users to perform transactions using both the credential and attached objects.
+zkLogin is a Sui primitive that let users send transactions from a Sui address using an OAuth credential, without publicly linking the two.
```
Comment (williamrobertson13) [inline doc/src/build/zk_login.md:5]:
```suggestion
zkLogin is a Sui primitive that lets users send transactions from a Sui address using an OAuth credential, without publicly linking the two.
```

File: doc/src/build/zk_login.md [hunk 4]
```diff
@@ -2,23 +2,25 @@
...
+1. **Optional Verified Identity**: However, a user can *opt in* to verify the OAuth identifier that was used to derive a particular Sui address. This serves as the foundation for a verifiable on-chain identity layer.
 
-1. **Foundation for the Identity Layer**: In the future, zkLogin can serve as an opt-in identity layer for users on-chain.
+1. **Accessible**: zkLogin is one of several native Sui signature schemes thanks to Sui's [crypto agility](https://docs.sui.io/devnet/learn/cryptography/sui-signatures). It smoothly integrates with other powerful Sui primitives like sponsored transactions and multisig.
 
-Are you a builder who wants to integrate with zkLogin into your wallet or application? Dive into our [Integration guide](#integration-guide).
+1. **Built with Rigor**. The [code](https://github.com/sui-foundation/zklogin-circuit) for zkLogin is open source and has been independently [audited](https://github.com/sui-foundation/zk-ceremony-client) by two firms specializing in zero knowledg. The public zkLogin ceremony for creating the common reference string attracted contributions from more than 100 participants.
```
Comment (williamrobertson13) [inline doc/src/build/zk_login.md:21]:
```suggestion
1. **Built with Rigor**. The [code](https://github.com/sui-foundation/zklogin-circuit) for zkLogin is open source and has been independently [audited](https://github.com/sui-foundation/zk-ceremony-client) by two firms specializing in zero knowledge. The public zkLogin ceremony for creating the common reference string attracted contributions from more than 100 participants.
```

File: doc/src/build/zk_login.md [hunk 5]
```diff
@@ -2,23 +2,25 @@
...
 ---
 
-zkLogin is a Sui primitive that lets wallets and apps link Sui objects with an OAuth credential—enabling users to perform transactions using both the credential and attached objects.
+zkLogin is a Sui primitive that let users send transactions from a Sui address using an OAuth credential, without publicly linking the two.
 
-zkLogin eliminates the need for users to handle private keys or recall mnemonics/passwords in wallets. By guiding users through the OAuth flow, an zkLogin address is generated, enabling any associated objects to execute transactions on-chain.
+zkLogin is designed with the following goals in mind:
 
-zkLogin is designed with the following principles in mind:
+1. **Simple Onboarding**: zkLogin allows users transact on Sui using the familiar OAuth login flow, eliminating the friction of handling cryptographic keys or remembering mnemonics.
```
Comment (williamrobertson13) [inline doc/src/build/zk_login.md:9]:
```suggestion
1. **Simple Onboarding**: zkLogin allows users to transact on Sui using the familiar OAuth login flow, eliminating the friction of handling cryptographic keys or remembering mnemonics.
```

File: doc/src/build/zk_login.md [hunk 6]
```diff
@@ -2,23 +2,25 @@
...
+1. **Accessible**: zkLogin is one of several native Sui signature schemes thanks to Sui's [crypto agility](https://docs.sui.io/devnet/learn/cryptography/sui-signatures). It smoothly integrates with other powerful Sui primitives like sponsored transactions and multisig.
 
-Are you a builder who wants to integrate with zkLogin into your wallet or application? Dive into our [Integration guide](#integration-guide).
+1. **Built with Rigor**. The [code](https://github.com/sui-foundation/zklogin-circuit) for zkLogin is open source and has been independently [audited](https://github.com/sui-foundation/zk-ceremony-client) by two firms specializing in zero knowledg. The public zkLogin ceremony for creating the common reference string attracted contributions from more than 100 participants.
+
+Are you a builder who wants to integrate with zkLogin into your application or wallet? Dive into our [Integration guide](#integration-guide).
 
 If you want to understand how zkLogin works, including how the zero-knowledge proof is generated, and how Sui verifies an zkLogin transaction, see [this section](#how-zklogin-works).
```
Comment (williamrobertson13) [inline doc/src/build/zk_login.md:25]:
```suggestion
If you want to understand how zkLogin works, including how the zero-knowledge proof is generated, and how Sui verifies a zkLogin transaction, see [this section](#how-zklogin-works).
```

File: doc/src/build/zk_login.md [hunk 7]
```diff
@@ -2,23 +2,25 @@
...
 
-1. **Foundation for the Identity Layer**: In the future, zkLogin can serve as an opt-in identity layer for users on-chain.
+1. **Accessible**: zkLogin is one of several native Sui signature schemes thanks to Sui's [crypto agility](https://docs.sui.io/devnet/learn/cryptography/sui-signatures). It smoothly integrates with other powerful Sui primitives like sponsored transactions and multisig.
 
-Are you a builder who wants to integrate with zkLogin into your wallet or application? Dive into our [Integration guide](#integration-guide).
+1. **Built with Rigor**. The [code](https://github.com/sui-foundation/zklogin-circuit) for zkLogin is open source and has been independently [audited](https://github.com/sui-foundation/zk-ceremony-client) by two firms specializing in zero knowledg. The public zkLogin ceremony for creating the common reference string attracted contributions from more than 100 participants.
+
+Are you a builder who wants to integrate with zkLogin into your application or wallet? Dive into our [Integration guide](#integration-guide).
```
Comment (williamrobertson13) [inline doc/src/build/zk_login.md:23]:
```suggestion
Are you a builder who wants to integrate zkLogin into your application or wallet? Dive into our [Integration guide](#integration-guide).
```

**Review by williamrobertson13 — COMMENTED** [review]
chiming in with some humble grammatical suggestions 😝

**Review by joyqvq — APPROVED** [review]
lgtm!

File: doc/src/build/zk_login.md (hunk 6 above)
Comment (randall-Mysten) [reply doc/src/build/zk_login.md]: or "how Sui verifies zkLogin transactions"

**Pull request merged**